apps/editing-toolkit/editing-toolkit-plugin/full-site-editing-plugin.php

```diff
@@ -261,17 +261,22 @@ function load_wpcom_block_editor_nux() {
 
 /**
  * Load editing toolkit block patterns
+ *
+ * @param obj $current_screen The current screen object.
  */
-function load_block_patterns() {
+function load_block_patterns( $current_screen ) {
 	if ( ! function_exists( '\gutenberg_load_block_pattern' ) ) {
 		return;
 	}
 
-	require_once __DIR__ . '/block-patterns/class-block-patterns.php';
+	if ( ! $current_screen->is_block_editor ) {
+		return;
+	}
 
+	require_once __DIR__ . '/block-patterns/class-block-patterns.php';
 	Block_Patterns::get_instance();
 }
-add_action( 'init', __NAMESPACE__ . '\load_block_patterns', 20 );
+add_action( 'current_screen', __NAMESPACE__ . '\load_block_patterns' );
```

Comment:
Noting here the loading of patterns only when viewing the block editor in wp-admin.

Reply:
I don't think `current_screen` is quite the right hook, it appears to fire on any admin screen including those not loading the block editor, e.g. `/wp-admin/tools.php` and `/wp-admin/users.php`.

Reply:
Edit: I wasn't sure about `current_screen` because I missed your check for `$current_screen->is_block_editor`. This looks good to me!